src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -161,18 +161,8 @@ public void Startup(StartupParams startupParams)
             PackageUploadBuilder.SetEngineVersion(startupParams.DynamoVersion);
             var uploadBuilder = new PackageUploadBuilder(dirBuilder, new MutatingFileCompressor());
 
-            // Align the package upload directory with the package download directory - 
-            // either the one selected by the user or the default directory.
-            string packageUploadDirectory;
-            if (startupParams.Preferences is PreferenceSettings preferences)
-            {
-                packageUploadDirectory = string.IsNullOrEmpty(preferences.SelectedPackagePathForInstall) ? 
-                    startupParams.PathManager.DefaultPackagesDirectory : preferences.SelectedPackagePathForInstall;
-            }
-            else
-            {
-                packageUploadDirectory = startupParams.PathManager.DefaultPackagesDirectory;
-            }
+            var packageUploadDirectory = startupParams.PathManager.DefaultPackagesDirectory;
```

Comment:
This logic is outdated and not needed anymore. `DefaultPackagesDirectory` will gives what we want in all cases.